packages/design-system/src/components/menu.tsx

```diff
@@ -26,7 +25,7 @@ export const itemCss = css(baseItemCss, {
   color: "$hiContrast",
   "&:focus, &[data-found]": {
     outline: "none",
-    backgroundColor: "$blue9",
+    backgroundColor: "$blue10",
```

Comment:
Same here, please let's stop using color tokens directly. Instead we should stick to semantic tokens like `background`, `text`, etc.